enterprise/lms_api.py

```diff
@@ -24,24 +26,36 @@
     CourseEnrollment = None
 
 
+LOGGER = logging.getLogger(__name__)
 LMS_API_DATETIME_FORMAT = '%Y-%m-%dT%H:%M:%SZ'
 
 
-class EnrollmentApiClient(object):
+class LmsApiClient(object):
```

Comment:
@pomegranited suggestion: make it abstract?

I have relatively negative experienec with `abc`, but it might be just the right use case for it.